### PR TITLE
CSI-5159 static provisioning

### PR DESCRIPTION
**Pull request by ArbelNathan**

### CODE REVIEW

File: controllers/utils/volumegroupcontent.go [hunk 1]
```diff
@@ -36,21 +36,21 @@ func AddMatchingPVToMatchingVGC(logger logr.Logger, client client.Client,
...
 		}
 		return nil, err
 	}
 
 	return vgc, nil
 }
 
-func CreateVolumeGroupContent(client client.Client, logger logr.Logger, vgcObj *volumegroupv1.VolumeGroupContent) error {
-	err := client.Create(context.TODO(), vgcObj)
+func CreateVolumeGroupContent(client client.Client, logger logr.Logger, vgClass *volumegroupv1.VolumeGroupContent) error {
```
Comment (matancarmeli7) [inline controllers/utils/volumegroupcontent.go:52]: the `vgClass` is a `volumegroupv1.VolumeGroupContent` type, can you change the var name?

Comment (ArbelNathan) [reply controllers/utils/volumegroupcontent.go]: done

**Review by matancarmeli7 — APPROVED** [review]
LGTM

**Review by ELENAGER — APPROVED** [review]
LGTM

**Pull request merged**